src/components/PDFView/index.js

```diff
@@ -52,6 +53,14 @@ class PDFView extends Component {
 
         const workerBlob = new Blob([pdfWorkerSource], {type: 'text/javascript'});
         pdfjs.GlobalWorkerOptions.workerSrc = URL.createObjectURL(workerBlob);
+        this.setMaxCanvaSizeInState = this.setMaxCanvaSizeInState.bind(this);
+        this.setMaxCanvaHeightInState = this.setMaxCanvaHeightInState.bind(this);
+        canvasSize.maxArea({
```

Comment:
Let's just call canvasSize.maxWidth instead, that way we could check if the file exceeds both width and height

Reply:
I can call canvasSize.maxWidth yes but I think I still need to call maxArea because there are cases where we don't surpass the max height and max width  but we surpass the max area

![image](https://github.com/Expensify/App/assets/19537677/6ecdc2a0-ca7f-4088-8216-1d061da61028)
For example here, if we had width 30k and height 30k, the area would be surpassed